public/sitemap.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+      xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+
+<url>
+  <loc>https://projectshut.vercel.app/</loc>
+  <lastmod>2023-05-26T14:19:26+00:00</lastmod>
+</url>
+
+
+</urlset>
```

Comment:
hey @krishnamangal98 we have a few pages such as the `projects` page, and `docs` page so ig we need to add sitemap for those pages as well